test/codeu/chat/server/DerbyStoreTest.java

```diff
@@ -26,7 +26,7 @@ public void doBefore() {
 
 	@Test
 	public void checkUserNameExists() {
-		final User user = controller.newUser("user");
+		final User user = controller.newUser("user", "test");
```

Comment:
Definitely. Also note that ideally you'd have at least two sets of tests: one set of tests for the low level database functionality, and another set of tests for the controller functionality.